Makefile

```diff
@@ -0,0 +1,21 @@
+.PHONY: test
+
+test:
+	docker-compose down
+	docker-compose build
+	docker-compose run --rm web bundle exec rails db:test:prepare
+	docker-compose run --rm web bundle exec rspec
```

Comment:
Should we run `bundle exec rake` here rather than just `rspec`, as it'll include Rubocop and ERB lint?

Reply:
Generally I've linted as part of a separate command (`make lint` or `make check`), as I find myself running the tests more often than the linting.

Maybe the linting is so quick I shouldn't bother separate the two.

I may have had some trouble running rake in this context, but I can't remember what it was - will look again, or @RobertM905 can have a go?